src/status_im2/constants.cljs

```diff
@@ -335,3 +335,30 @@
 (def ^:const auth-method-biometric "biometric")
 (def ^:const auth-method-biometric-prepare "biometric-prepare")
 (def ^:const auth-method-none "none")
+
+(def ^:const int->weekday
+  "Returns the corresponding string representation of a weekday
```

Comment:
Nitpick: even though maps can be used as functions, the docstring could start with `Maps the corresponding...` instead of `Returns ...`.